__fixtures__/circular/pnpm-lock.yaml

```diff
@@ -8,28 +8,28 @@ dependencies:
 packages:
 
   /d@1.0.0:
-    resolution: {integrity: sha1-dUu1v+VUUdpppYuU1F9MWwRi1Y8=}
+    resolution: {integrity: sha512-9x1NruMD5YQ7xccKbGEy/bjitRfn5LEIhJIXIOAXC8I1laA5gfezUMVES1/vjLxfGzZjirLLBzEqxMO2/LzGxQ==}
```

Comment:
Most `pnpm-lock.yaml` files in `__fixtures__` could be updated by running `pnpm dedupe`. There were a few lockfiles using older formats that I had to update manually by copying the checksum from `pnpm view <name>@<version>`.